app/views/potential_matches/_in_progress.html.erb

```diff
@@ -5,12 +5,12 @@
 </p>
 
 <p>
-  <%= ts("If the progress value below doesn't change when you reload the page 
+  <%= ts("If the progress value below doesn't change when you reload the page
           after about 10 minutes, there may have been a problem. Please check again after an hour, and then contact archive support.") %>
 </p>
 
 <p>
   <%= ts("Potential match generation progress:") %> <strong><%= @progress %>%</strong>
 </p>
 
-<p class="actions" role="button"><%= link_to ts("Cancel Potential Match Generation"), cancel_generate_collection_potential_matches_path(@collection) %></p>
+<span class="actions" role="button"><%= button_to ts("Cancel Potential Match Generation"), cancel_generate_collection_potential_matches_path(@collection), method: :post %></span>
```

Comment:
I'm a little torn on this. The W3 HTML validator flags this because `form` isn't a valid child of `span`. However, it at least displays correctly, which is more than I can say for the `p` tag -- my browser closes before the `form` elements starts, and the HTML validator tells me it can't find the opening `p` tag. 

I guess we can leave it like this for now and rethink the page a bit later. However, we should take the `role` off the span.